Pub-Servers.md

```diff
@@ -1,8 +0,0 @@
-This is an informal registry of pub servers. Ping the pub-owners for an invite.
```

Comment:
there's nothing in these docs about how to find and join an existing pub server, as most people would want to do. i see this is in the [scuttlebot](https://github.com/ssbc/scuttlebot) repo, but then i'm really confused as to which is the preferred source of truth.